test/system/010-images.bats

```diff
@@ -370,6 +370,13 @@ EOF
     run_podman --root $imstore/root rmi --all
 }
 
+@test "podman images with retry" {
+    run_podman pull -q --retry 4 --retry-delay "10s" $IMAGE
+    run_podman 125 pull -q --retry 4 --retry-delay "foobar" $IMAGE
+    is "$output" 'Error: time: invalid duration "foobar"' "bad retry-delay"
+    run_podman rmi --all
```

Comment:
Eek, please no, that will force a subsequent re-fetch, which will then introduce flakes